[2023-01-22T03:50:07.197+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T03:50:01.546804+00:00 [queued]>
[2023-01-22T03:50:07.211+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T03:50:01.546804+00:00 [queued]>
[2023-01-22T03:50:07.212+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-22T03:50:07.213+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-01-22T03:50:07.214+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-22T03:50:07.230+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): create_encrypt_file> on 2023-01-22 03:50:01.546804+00:00
[2023-01-22T03:50:07.238+0000] {standard_task_runner.py:55} INFO - Started process 1780 to run task
[2023-01-22T03:50:07.243+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'debts_tracker_etl', 'create_encrypt_file', 'manual__2023-01-22T03:50:01.546804+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/debts_tracker_etl.py', '--cfg-path', '/tmp/tmpfjk351b4']
[2023-01-22T03:50:07.247+0000] {standard_task_runner.py:83} INFO - Job 40: Subtask create_encrypt_file
[2023-01-22T03:50:07.343+0000] {task_command.py:388} INFO - Running <TaskInstance: debts_tracker_etl.create_encrypt_file manual__2023-01-22T03:50:01.546804+00:00 [running]> on host 4b1c480b6ccb
[2023-01-22T03:50:07.432+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Tony Stark, Peter Parker
AIRFLOW_CTX_DAG_ID=debts_tracker_etl
AIRFLOW_CTX_TASK_ID=create_encrypt_file
AIRFLOW_CTX_EXECUTION_DATE=2023-01-22T03:50:01.546804+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-22T03:50:01.546804+00:00
[2023-01-22T03:50:07.460+0000] {logging_mixin.py:137} INFO - 0-----
[2023-01-22T03:50:07.461+0000] {logging_mixin.py:137} INFO - b'gAAAAABjzLJvyC46v23l8K6iadkoABw0H58IlSkYWrLzJ04lwT3Lf4gm5cucq_14VGy8pP4U8ujmjNGEtlYGCN1JrMscYu5ixAJIyloD84mq_m_e8AdVQ0X9mImQlhRrvjxi6rHN96jN2ar9Ak5r9v5f4WBSPlwPV_TypD66JDP0_bMlOgfYHkPnwzKhUj5xdo6Rxfp2R4PYy7154hPWkfwbvIiBxM_CDvnZNiCI0HPQx82eNATYISixA6oOD8wCLRc8HHY5mjJ1uQfPRjo__0ea4PDeisGDk70SsPXhAcPu7Q8BEcVb-aSGwjhm9MvIABHszN9opeXBGJiTcZtIxUXk5pG1s0FbqSbwqMmtlxKxcBB0Acbvaq8='
[2023-01-22T03:50:07.462+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-22T03:50:07.475+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=debts_tracker_etl, task_id=create_encrypt_file, execution_date=20230122T035001, start_date=20230122T035007, end_date=20230122T035007
[2023-01-22T03:50:07.538+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-01-22T03:50:07.558+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
